### PR TITLE
Update 103-data-transformation.md

### PR DESCRIPTION
**Pull request by Wblake95**
Descriptive article that breaks down the process.
Additionally, links to examples in Python and R

### CODE REVIEW

File: src/data/roadmaps/data-analyst/content/107-data-cleaning/103-data-transformation.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
...
\ No newline at end of file
+Data Transformation, also known as Data Wrangling, is an essential part of a Data Analyst's role. This process involves the conversion of data from a raw format into another format to make it more appropriate and valuable for a variety of downstream purposes such as analytics. Data Analysts transform data to make the data more suitable for analysis, ensure accuracy, and to improve data quality. The right transformation techniques can give the data a structure, multiply its value, and enhance the accuracy of the analytics performed by serving meaningful results.
+
+- [@article@What is Data Transformation - Geeks for Geeks](https://www.geeksforgeeks.org/what-is-data-transformation/)
```
Comment (dansholds) [inline src/data/roadmaps/data-analyst/content/107-data-cleaning/103-data-transformation.md:5]:
```suggestion
Data Transformation, also known as Data Wrangling, is an essential part of a Data Analyst's role. This process involves the conversion of data from a raw format into another format to make it more appropriate and valuable for a variety of downstream purposes such as analytics. Data Analysts transform data to make the data more suitable for analysis, ensure accuracy, and to improve data quality. The right transformation techniques can give the data a structure, multiply its value, and enhance the accuracy of the analytics performed by serving meaningful results.

Learn more from the following resources:

- [@article@What is Data Transformation?](https://www.geeksforgeeks.org/what-is-data-transformation/)
```

Comment (dansholds) [reply src/data/roadmaps/data-analyst/content/107-data-cleaning/103-data-transformation.md:5]: You mention in your PR description that you've added links to Python and R examples but I don't see them?

**Review by dansholds — CHANGES_REQUESTED** [review]

**Review by Wblake95 — COMMENTED** [review]
Sorry for my mistake

File: src/data/roadmaps/data-analyst/content/107-data-cleaning/103-data-transformation.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
```
Comment (dansholds) [inline src/data/roadmaps/data-analyst/content/107-data-cleaning/103-data-transformation.md:5]:
```suggestion
- [@article@What is Data Transformation](https://www.tibco.com/glossary/what-is-data-transformation)
```

File: src/data/roadmaps/data-analyst/content/107-data-cleaning/103-data-transformation.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
...
+- [@article@Example with Python](https://www.geeksforgeeks.org/time-series-data-transformation-using-python/)
```
Comment (dansholds) [inline src/data/roadmaps/data-analyst/content/107-data-cleaning/103-data-transformation.md:6]:
```suggestion
- [@article@Transforming Data with Python](https://www.udacity.com/blog/2023/01/transforming-data-with-python.html)
```

File: src/data/roadmaps/data-analyst/content/107-data-cleaning/103-data-transformation.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
...
+- [@article@Example with R](https://www.geeksforgeeks.org/data-cleaning-transformation-with-dplyr-in-r/)
```
Comment (dansholds) [inline src/data/roadmaps/data-analyst/content/107-data-cleaning/103-data-transformation.md:7]:
```suggestion
- [@article@Data Transformation with R](https://r4ds.had.co.nz/transform.html)
```

**Review by dansholds — CHANGES_REQUESTED** [review]
I've found interesting alternative links to the ones you have, we already have a lot of links to this site throughout roadmap.sh so it's nice to expose our users to other sites and creators 😄

**Comment by Wblake95** [comment]
Those sound great!  I have never heard of tibco or r4ds.

**Review by dansholds — APPROVED** [review]

**Pull request merged**